podimo/client.py

```diff
@@ -48,6 +49,7 @@ def generateHeaders(self, authorization):
         return gHdrs(authorization, self.locale) 
 
     async def post(self, headers, query, variables, scraper):
+        scraper = ZenRowsClient(ZENROWS_API)
```

Comment:
Would it be possible to make this optional? i.e. when `ZENROWS_API` is not set/is None, then this line is skipped and the normal scraper is used